internal/resources/managementclusterregistration/management_cluster_registration_spec_flatten_test.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package managementclusterregistration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	clustermodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/cluster"
+	managementclusterregistrationmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/managementclusterregistration"
+)
+
+func TestFlattenSpec(t *testing.T) {
```

Comment:
Are you not using the converter for api model-> schema conversion?

Reply:
This schema is already pretty easy and the conversion is quite short too. With more complicated schema in the future I will use the converter.